src/components/selector-dropdown/SelectorDropdown.js

```diff
@@ -74,29 +76,13 @@ class SelectorDropdown extends React.Component<Props, State> {
 
     setActiveItemID = (id: string | null) => {
         this.setState({ activeItemID: id });
-        this.scrollItemIntoView(id);
+        scrollItemIntoView(id, OVERLAY_WRAPPER_CLASS);
     };
 
     listboxID: string;
 
     selectorDropdownRef: { current: null | HTMLDivElement };
 
-    scrollItemIntoView = (id: string | null) => {
-        // @NOTE: breaks encapsulation but alternative is unknown child ref
-        const itemEl = id ? document.getElementById(id) : null;
-        if (itemEl) {
-            let parentEl = itemEl;
-            while (
-                parentEl.parentElement instanceof HTMLElement &&
-                !parentEl.classList.contains('scroll-wrap-container') && // Scroll the container, not the body when contained in a scroll-container.
-                !parentEl.classList.contains('modal') // Scroll the modal, not the body when contained in a modal.
```

Comment:
Do we no longer cover the modal use case?

Reply:
I will put the modal use case back. When I tested the functionality it worked for MetadataInstanceEditor but I didn't test for the modal case in EUA